signac/_synced_collections/backends/collection_json.py

```diff
@@ -116,6 +117,16 @@ def json_attr_dict_validator(data):
 """
 
 
+class _IsWindows:
+    """A truthy type that is False on Windows and True otherwise."""
```

Comment:
Shouldn't it be the other way round? "_IsWindows" should be `True` on Windows and `False` otherwise?

Reply:
See below.